### DISCUSSION TITLE
Benefit of ssr false + adapter-node?

### DISCUSSION THREAD

**Discussion by fabiot21** [sveltejs/kit#7116] [Q&A] [answered]
In practice, what would be a case where is better to disable ssr having the node adapter (not adapter-static for SPA because i need a backend for endpoints)? Any advantage?

Thanks in advance!

**Comment by david-plugge** [comment, marked as answer]
An admin dashboard for example.
If you don't need seo and don't care about looking at a white screen until the page is fully loaded.

I'm currently working on an admin dashboard and ssr disabled is really nice if you are using a text editor for example that uses browser only apis or the window object. You simply don't have to take care of such situations.
